doc/glossary.rst

```diff
@@ -287,6 +287,19 @@
 	Object Storage Device
                 See :term:`OSD`.
 
+        OMAP
+                "object map". A key-value store (a database) that is used to
+                reduce the time it takes to read data from and to write to the
```

Comment:
s/and to/and/

Reply:
I'm going to leave the prepositional phrase intact. It makes the sentence less ambiguous, and more legible for non-native English readers.